src/Bookings.js

```diff
@@ -1,17 +1,24 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
 import SearchButton from "./Search.js";
```

Comment:
Could you rename this import to just `Search`, otherwise it's a bit confusing.

Reply:
Done